Documentation/building/windows-instructions.md

```diff
@@ -103,13 +93,8 @@ For advanced debugging using WinDBG see [Debugging CoreFX on Windows](https://gi
 
 ### Notes
 * At any given time, the corefx repo might be configured to use a more recent compiler than
-the one used by the most recent Visual Studio IDE release.  This means the corefx codebase might
+the one used by the installed .NET Core SDK. This means the corefx codebase might
```

Comment:
I think it is worth adding a pointer to the SDK we use to build so folks can know which version they need. For now that is just to look at the DotnetCLIVersion.txt file but will change to be in global.json once we finish our engineering transition.